content/php/concepts/math-functions/terms/deg2rad/deg2rad.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: 'deg2rad()'
+Description: 'Converts the number in degrees to the radian equivalent'
+Subjects:
+  - 'Computer Science
+  - 'Web Development
+  - 'Math Functions'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`deg2rad()`** function converts the number in degrees to the radian equivalent.
+
+The result returns a value type of `float`.
```

Comment:
Can we move this line to the end of the syntax section? (Inserted before line 28)

Reply:
Done